filetests/isa/x86/legalize-i64.clif

```diff
@@ -25,3 +25,25 @@ ebb0(v1: i64, v2: i64):
     ; nextln: v10 = iconcat $v10_lsb, $v10_msb
     return v10
 }
+
+function %icmp_eq(i64, i64) -> b1 {
+ebb0(v1: i64, v2: i64):
+    v10 = icmp eq v1, v2
+    ; check: v1 = iconcat $(v1_lsb=$V), $(v1_msb=$V)
+    ; nextln: v2 = iconcat $(v2_lsb=$V), $(v2_msb=$V)
+    ; nextln: $(v10_lsb=$V) = icmp eq $v1_lsb, $v2_lsb
+    ; nextln: $(v10_msb=$V) = icmp eq $v1_msb, $v2_msb
+    ; nextln: v10 = band $v10_lsb, $v10_msb
+    return v10
+}
+
+function %icmp_ne(i64, i64) -> b1 {
```

Comment:
Can you also add tests for one of the other condition codes, please?

Reply:
On it :)